### PR TITLE
added tiggers for p,q,r,w

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

File: plugins/aws/redshift/auditLoggingEnabled.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/redshift/latest/mgmt/db-auditing-console.html',
     recommended_action: 'Modify Redshift clusters to enable audit logging',
     apis: ['Redshift:describeClusters', 'Redshift:describeLoggingStatus', 'STS:getCallerIdentity'],
+    realtime_triggers: ['redshift:CreateCluster', 'redshift:EditLogging'],
```
Comment (AkhtarAmir) [inline plugins/aws/redshift/auditLoggingEnabled.js:13]: restoreFromClusterSnapshot

File: plugins/aws/route53/domainAutoRenew.js [hunk 2]
```diff
@@ -9,6 +9,8 @@ module.exports = {
     link: 'http://docs.aws.amazon.com/Route53/latest/APIReference/api-enable-domain-auto-renew.html',
     recommended_action: 'Enable auto renew for the domain',
     apis: ['Route53Domains:listDomains'],
+    realtime_triggers: ['route53domains:RegisterDomain','route53domains:EnableAutoRenew'],
```
Comment (AkhtarAmir) [inline plugins/aws/route53/domainAutoRenew.js:12]: DisableDomainAutoRenew

File: plugins/aws/route53/domainTransferLock.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     link: 'http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-transfer-from-route-53.html',
     recommended_action: 'Enable the transfer lock for the domain',
     apis: ['Route53Domains:listDomains'],
+    realtime_triggers: ['route53domains:RegisterDomain', 'route53Domain:EnableDomainTransferLock'],
```
Comment (AkhtarAmir) [inline plugins/aws/route53/domainTransferLock.js:12]: disableDomainTransferLock

**Review by AkhtarAmir — CHANGES_REQUESTED** [review]

File: plugins/aws/rds/rdsCmkEncryptionEnabled.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html',
     recommended_action: 'RDS does not currently allow modifications to encryption after the instance has been launched, so a new instance will need to be created with KMS CMK encryption enabled.',
     apis: ['RDS:describeDBInstances', 'KMS:listAliases'],
+    realtime_triggers: ['rds:CreateDBInstance', 'rds:RestoreDBInstanceFromDBSnapshot'],
```
[the remark this answers is not in the document]
Comment (fatima99s) [reply plugins/aws/rds/rdsCmkEncryptionEnabled.js:13]: we cant add the cmk encryption by modifying the db-instance

**Review by AkhtarAmir — APPROVED** [review]

**Review by alphadev4 — APPROVED** [review]

**Pull request merged**